app/AccountCreation.tsx

```diff
@@ -0,0 +1,106 @@
+import { ReactElement, useState } from "react";
+import { Button, Form } from "react-bootstrap";
+import { userNameAlreadyExists } from "@/server/getCredentials";
+import styled from "styled-components";
+import { PageHeader } from "./ui/PageHeader";
+import { FormGroup } from "./ui/FormGroup";
+import { FormControl } from "./ui/FormControl";
+
+export function AccountCreation(props: {
+  backToLogin: () => void;
+}): ReactElement {
+  const { backToLogin } = props;
+  const [errorMessage, setErrorMessage] = useState("");
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  return (
+    <AccountCreationStyling>
+      {errorMessage}
+      <div style={{ height: "20px" }}></div>
```

Comment:
is this for spacing? use margins

Reply:
https://github.com/auberginearms/starterApp/pull/4/commits/960f2c9f8409d097615695a9adca3d1f38100813